pkg/project/resource_project_repo.go

```diff
@@ -156,13 +166,14 @@ var deleteRepos = func(projectKey string, repoKeys []RepoKey, m interface{}, g *
 var deleteRepo = func(projectKey string, repoKey RepoKey, m interface{}) error {
 	log.Println("[DEBUG] deleteRepo")
 
-	_, err := m.(*resty.Client).R().
+	_, err := cloneResty(m.(*resty.Client)).
```

Comment:
And here you're copy and pasting the same code? And, it doesn't seem to you like you shouldn't?